src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -51,6 +57,12 @@ module.exports = class ArtworkCommercialView extends Backbone.View
     # Show the new buy now flow if you have the lab feature enabled
     if loggedInUser?.hasLabFeature('New Buy Now Flow')
 
+      analytics.track('Click', {
+        subject: 'buy now',
+        type: 'button',
+        flow: 'buy'
```

Comment:
should the flow here be `buy now`?

Reply:
Oh, good catch! Yes, it should be `buy now`